### DISCUSSION TITLE
NRK Tegnspråk [NO]

### DISCUSSION THREAD

**Discussion by POLARIS183** [iptv-org/community#863] [Q&A]
### Channel ID

NRKTegnsprak.no

### Notes

Norwegian sign language station - often simulcasts NRK1 with NSL overcast.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19841
